source/appModules/winword.py

```diff
@@ -11,10 +11,20 @@
 import appModuleHandler
 from scriptHandler import script
 import ui
+from logHandler import log
 from NVDAObjects.IAccessible.winword import WordDocument as IAccessibleWordDocument
 from NVDAObjects.UIA.wordDocument import WordDocument as UIAWordDocument
 from NVDAObjects.window.winword import WordDocument
 from utils.displayString import DisplayStringIntEnum
+from typing import TYPE_CHECKING
+
+if TYPE_CHECKING:
+	import inputCore
+
+
+# From WdOutlineLevel enumeration
+# See https://learn.microsoft.com/fr-fr/office/vba/api/word.wdoutlinelevel
+wdOutlineLevelBodyText = 10
```

Comment:
This either needs to be marked const
```suggestion
WD_OUTLINE_LEVEL_BODY_TEXT: int = 10
```
Or refactored to be an enum for future extensibility
```suggestion
class WdOutlineLevel(IntEnum):
	BODY_TEXT = 10
```